pkg/sql/executor_ir.go

```diff
@@ -59,7 +59,7 @@ func RunSQLProgram(sqlProgram string, db *DB, modelDir string, session *pb.Sessi
 }
 
 func runSQLProgram(wr *PipeWriter, sqlProgram string, db *DB, modelDir string, session *pb.Session) error {
-	sqls, err := SplitMultipleSQL(sqlProgram)
+	sqls, err := parse(db.driverName, sqlProgram)
```

Comment:
`parse` and below `newParser().Parse(sql)` seems confusing, and the filename `parse.go` and `parser.go` also confusing, a better name is needed.

Reply:
I have renamed `newParser().Parse(sql)` to `newExtendedSyntaxParser().Parse(sql)`.